content/sensu-go/5.18/installation/install-sensu.md

```diff
@@ -15,7 +15,7 @@ menu:
 - [Install sensuctl](#install-sensuctl)
 - [Install Sensu agents](#install-sensu-agents)
 - [Commercial features](#commercial-features)
-- [Next steps](#next-steps)
+- [Next step: Generate certificates](#next-step-generate-certificates)
 
 This installation guide describes how to install the Sensu backend, Sensu agent, and sensuctl command line tool.
 
```

Comment:
In Install content, need to add callouts that say "If you're doing this for anything beyond POC or devel env, we recommend securing with TLS. See the Generate Certificates guide > Secure > Cluster" (this should set them on the path). Need an alternate next step for people who don't want to set up a cluster.